imap_processing/codice/codice_l1a.py

```diff
@@ -479,10 +493,10 @@ def get_params(packet: space_packet_parser.parser.Packet) -> tuple[int, int, int
     view_id : int
         Provides information about how data was collapsed and/or compressed.
     """
-    table_id = packet.data["TABLE_ID"].raw_value
-    plan_id = packet.data["PLAN_ID"].raw_value
-    plan_step = packet.data["PLAN_STEP"].raw_value
-    view_id = packet.data["VIEW_ID"].raw_value
+    table_id = packet.table_id.data[0]
```

Comment:
Are you able to remove any of the `[0]` indexes now and keep the entire array as you go through? I think we'd want each epoch associated with its own table_id and not just the first one.